jsonnet/components/admission-webhook.libsonnet

```diff
@@ -168,4 +168,42 @@ function(params)
         },
       ],
     },
+    networkPolicy: {
+      apiVersion: 'networking.k8s.io/v1',
+      kind: 'NetworkPolicy',
+      metadata: {
+        annotations: {
+          'include.release.openshift.io/hypershift': 'true',
+          'include.release.openshift.io/ibm-cloud-managed': 'true',
+          'include.release.openshift.io/self-managed-high-availability': 'true',
+          'include.release.openshift.io/single-node-developer': 'true',
+        },
+        name: 'prometheus-operator-access',
+        namespace: 'openshift-monitoring',
+      },
+      spec: {
+        podSelector: {
+          matchLabels: {
+            'app.kubernetes.io/name': 'prometheus-operator',
+          },
+        },
+        policyTypes: [
+          'Ingress',
+          'Egress',
+        ],
+        ingress: [
+          {
+            ports: [
+              {
+                port: '8443',
+                protocol: 'TCP',
+              },
+            ],
+          },
+        ],
+        egress: [
```

Comment:
I'm not sure that thee admission webhook needs any egress.

Reply:
this allows curl admission webhook enpoint from any pod of any space